rails/db/migrate/20180427132538_add_flag_to_developments.rb

```diff
@@ -0,0 +1,5 @@
+class AddFlagToDevelopments < ActiveRecord::Migration[5.1]
+  def change
+    add_column :developments, :flag, :boolean
```

Comment:
Supplementary state booleans should have default values set to express their initial state.

Reply:
I agree that this should have a default value on the grounds that it can be confusing (or undocumented) if we start assigning a meaning to nil. If we want to be really robust, we might consider a "reports" table with a record of each flag, for what development, who flagged it, who approved it, and why.

Reply:
Ok I pushed an update to `NOT NULL` the column and set the default to false. I feel like this is a solid MVP for now but that eventually, this feature may converge with the suggested edits feature.